tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -87,34 +96,34 @@ IncludeByKey.DEEPBUFFER_FW_DMA_MS{
 #
 #SSP Index: 0, Direction: duplex
 Object.Dai {
-	SSP."0" {
+	SSP.1 {
```

Comment:
does this index need to change? I understood from @ranj063 's explanation that this ."0" was just a way to make the object unique. And if it does need to change, could it be a separate patch because it should have to bearing on the result and no relationship with the headset/amplifier changes.

Reply:
does not need to change